src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tBody = table.querySelector('tbody');
+
+const sortByText = (rowsArray, index) => {
+  rowsArray.sort((a, b) => a.cells[index].innerText
+    .localeCompare(b.cells[index].innerText));
+};
+
+const sortByAge = (rowsArray, index) => {
+  rowsArray.sort((a, b) => +a.cells[index].innerText
+    - +b.cells[index].innerText);
+};
+
+const sortBySalary = (rowsArray, index) => {
+  rowsArray.sort((a, b) =>
+    +a.cells[index].innerText.slice(1).split(',').join('')
+    - +b.cells[index].innerText.slice(1).split(',').join('')
```

Comment:
Would be great to create a function helper for normalizing the salary. Now it looks a bit complicated and hard to read